site/docs/latest/deployment/instance.md

```diff
@@ -141,26 +137,24 @@ serviceUrl=http://pulsar.us-west.example.com:8080/
 
 ## Provisioning new tenants
 
-Pulsar was built as a fundamentally {% popover multi-tenant %} system. New tenants can be provisioned as Pulsar {% popover properties %}. Properties can be
+Pulsar was built as a fundamentally {% popover multi-tenant %} system.
 
-To allow a new tenant to use the system, we need to create a new {% popover property %}. You can create a new property using the [`pulsar-admin`](../../reference/CliTools#pulsar-admin-properties-create) CLI tool:
+To allow a new {% popover tenant %} to use the system, we need to create a new one. You can create a new tenant using the [`pulsar-admin`](../../reference/CliTools#pulsar-admin-tenants-create) CLI tool:
 
 ```shell
-$ bin/pulsar-admin properties create test-prop \
+$ bin/pulsar-admin tenants create test-prop \
```

Comment:
Since `prop` might not make a lot of sense to reader, we should probably switch to `test-tenant`?